docs/themes/thxvscode/layouts/partials/apiNav.html

```diff
@@ -80,9 +83,16 @@
     </ul>
     <hr/>
     <ul class="nav" id="secondary-nav">
-        <li {{ if $in_api_section }} class="active" {{ end }}>
-            <a class="docs-home" href="/docs/apis/" {{ if $in_api_section }}
-                aria-label="Current Page: API Overview" {{ end }}>API Overview</a>
+        <li>
+            <select name="API version" id="api-version" onchange="window.location.href = this.value;">
+                {{ $selectedApiVersion := trim $current.RelPermalink "/" }}
+                {{ $selectedApiVersion = index (split $selectedApiVersion "/") 2 }}
+                {{ $isV2 := eq $selectedApiVersion "2.0"}}
```

Comment:
Same as my comment [above](https://github.com/microsoft/FluidFramework/pull/18710/files#r1421097271) - we need this logic to be dynamic based on the contents of `versions.json`. Otherwise, we'll have to modify this code every time we release a new version or stop supporting an old version. We'd like to control that strictly from the versions file.

Reply:
Up-to-date